### DISCUSSION TITLE
Separate Chapter from Prima

### DISCUSSION THREAD

**Discussion by mbab** [DivinumOfficium/divinum-officium#3800] [Ideas]
At least Ordo Preadicatorum allows to read it after Laudes or Prima.

Should we separate it for other versions? (Put it in Horas menu?)

**Comment by Augustinus-Altovadensis** [comment]
The Cistercians also usually read it after Prima, they even leave the choir and go to the Chapter Hall.
I personally have it as a part of Prima, but separating would work as well.
